### PR TITLE
Quiz question answer

### PR DESCRIPTION
**Pull request by Shailseen**
## Files to be review
- Question-context
- Question-card
- Answer-Card
- Reducer for set api address
- Question-tracker 
- Score-card
**[Preview Link](https://deploy-preview-2--chic-klepon-841b4a.netlify.app/)**

### CODE REVIEW

File: src/Contexts/question-context.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          answer: item.correct_answer,
+          option: [item.correct_answer],
+          userAnswer: "",
+        };
+        item.incorrect_answers.map((item) => obj.option.push(item));
+        obj.option.sort();
+        temp.push(obj);
+      });
+      setQuestionData(temp);
+      console.log(temp);
```
Comment (anupkgurung) [inline src/Contexts/question-context.js]: you can remove log and comments if not required.

File: src/components/AnswerCard/AnswerCard.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+              <div
+                className={classNames(
+                  styles.option_container,
+                  userAnswer === ""
+                    ? styles.no_change
+                    : answer === userAnswer && item === userAnswer
+                    ? styles.selected
+                    : item !== answer && userAnswer === item
+                    ? styles.non_selected
+                    : styles.no_change
```
Comment (anupkgurung) [inline src/components/AnswerCard/AnswerCard.jsx:21]: ternary nesting can be confusing sometimes. It's basically used as a one liner

File: src/components/QuestionTracker/QuestionTracker.module.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  height: 1.3rem;
+}
+
+.non_attempted {
+  width: 100%;
+  height: 1.3rem;
+}
+
+.tracker_container {
+  background-color: rgba(255, 255, 255, 0.15);
```
Comment (anupkgurung) [inline src/components/QuestionTracker/QuestionTracker.module.css]: color should be consistent either use rgb or variables or hex. preferred would be hex

File: src/routes/QuestionList/QuestionList.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      </div>
+      <div
+        className={classNames(
+          styles.question_wrapper,
+          second === 0 && minute === 0 && styles.display_none
+        )}
+      >
+        <div className={styles.flex}>
+          <div>
+            Q. {questionIndex + 1}/{questionData && questionData.length}
```
Comment (anupkgurung) [inline src/routes/QuestionList/QuestionList.jsx]: you could use template literal here.

**Review by anupkgurung — APPROVED** [review]
left few suggestion.
rest look good

**Review by arpit01923 — COMMENTED** [review]
- good folder structure
- try barrel exports and absolute paths
- questions in UI has some problems
- after 1 minute it will show the result, but you have to show it after 5 minutes

**Pull request merged**